perf-cluster-operation/main.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This is a tool, it should be under /tools. Also, a shorter name would be nice. What about "perf-tool", "perf-ops", or something like that?

Reply:
Moved to `tools/perf-ops`. Thanks.